pkg/controlplane/instance.go

```diff
@@ -225,8 +225,8 @@ type EndpointReconcilerConfig struct {
 	Interval   time.Duration
 }
 
-// Master contains state for a Kubernetes cluster master/api server.
-type Master struct {
+// Instance contains state for a Kubernetes cluster api server instance.
```

Comment:
Looks like this needs to be reflected in https://github.com/kubernetes/kubernetes/blob/66334f02e8c520df7973c397246da82cd4db2769/pkg/controlplane/instance_test.go#L202 also. Might be worth also updating var names there as well?